tce/src/applibs/Simulator/SimControlLanguageCommand.cc

```diff
@@ -722,6 +722,75 @@ SimControlLanguageCommand::verifyBreakpointHandles(
     return true;
 }
 
+/**
+ * Sets corresponding memory address and address space given by address string.
```

Comment:
Should the word 'parse' used here? Parses the address tring to corresponding memory address and address space or similar.